charts/argo-cd/templates/argocd-application-controller/clusterrolebinding.yaml

```diff
@@ -13,4 +13,12 @@ subjects:
 - kind: ServiceAccount
   name: {{ include "argo-cd.controller.serviceAccountName" . }}
   namespace: {{ include "argo-cd.namespace" . }}
+{{- $namespaces := index .Values.configs.params "applicationsetcontroller.namespaces" -}}
```

Comment:
Though I was distracted about the logic itself,  the original issue ( https://github.com/argoproj/argo-helm/issues/2919 ) is about `argocd-applicationset-controller` sa, not `argocd-application-controller`sa.
Are you sure that here is the right place to add code?

Reply:
Oh. I was mistaken. I'll close the PR, and I'll work on the argocd-applicationset-controller with the same logic and post the PR again. Thanks for the attention.